frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -29,6 +29,7 @@ action.GitMachete.GitCommandUpdatingCurrentBranchBackgroundable.notification.tit
 
 # Quasi-title capitalization intended since we always write "Git Machete" with initial caps.
 action.GitMachete.GitMacheteRepositoryUpdateBackgroundable.task-title=Updating Git Machete status
+action.GitMachete.GitMacheteRepositoryUpdateBackgroundable.notification.title.failed=Cannot display Git Machete branch layout
```

Comment:
Maye `Cannot create Git Machete branch layout` would be more appropriate?

Reply:
Done